From 41522cc7f21de2e2d89a8b4af9f0e74cb7c11f1b Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Thu, 14 May 2015 09:13:49 -0400
Subject: [PATCH 764/933] Fix: build against Ubuntu kernels

Recent Ubuntu kernels appear to define DEBIAN_API_VERSION, but they are
not really a Debian kernel per-se: the abi-debian-version.sh cannot find
the expected Debian kernel version string layout.

Therefore, change the conditional check to use directly the
LTTNG_DEBIAN_VERSION_CODE define, which is defined only when the
abi-debian-version.sh script figures out it builds against a Debian
kernel.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
---
 drivers/staging/lttng/lttng-kernel-version.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/lttng/lttng-kernel-version.h b/drivers/staging/lttng/lttng-kernel-version.h
index 59e19df..8933c31 100644
--- a/drivers/staging/lttng/lttng-kernel-version.h
+++ b/drivers/staging/lttng/lttng-kernel-version.h
@@ -61,7 +61,7 @@
 
 #define LTTNG_DEBIAN_KERNEL_RANGE(a_low, b_low, c_low, d_low, e_low, f_low, \
 		a_high, b_high, c_high, d_high, e_high, f_high) \
-	(defined(DEBIAN_API_VERSION) && \
+	(defined(LTTNG_DEBIAN_VERSION_CODE) && \
 		LTTNG_DEBIAN_VERSION_CODE >= \
 		LTTNG_DEBIAN_KERNEL_VERSION(a_low, b_low, c_low, d_low, e_low, f_low) && \
 		LTTNG_DEBIAN_VERSION_CODE < \
-- 
2.0.2

